docs/source/json-api/index.rst

```diff
@@ -1053,25 +1053,21 @@ JavaScript/Node.js example demonstrating how to establish Streaming API connecti
 
 .. code-block:: javascript
 
-    const WebSocket = require("ws")
```

Comment:
you need it when you run it from command line, the line above mentions Node.js BTW:
> JavaScript/Node.js example demonstrating how to establish Streaming API connection:

Remove Node.js and say something like In-browser JavaScript.